.editorconfig

```diff
@@ -44,3 +44,24 @@ dotnet_style_parentheses_in_arithmetic_binary_operators = never_if_unnecessary:s
 dotnet_style_parentheses_in_other_binary_operators = never_if_unnecessary:silent       #Prefer var v = a || b && c; Over var v = a || (b && c);
 dotnet_style_parentheses_in_other_operators = never_if_unnecessary:error               #Prefer var v = a.b.Length; Over var v = (a.b).Length;
 dotnet_style_parentheses_in_relational_binary_operators = always_for_clarity:error     #Prefer var v = (a < b) == (c > d); Over var v = a < b == c > d;
+
+# Modifier preferences
+dotnet_style_require_accessibility_modifiers = omit_if_default:silent  #Prefer accessibility modifiers to be specified except if they are the default modifier.
+
+# Expression-level preferences
+dotnet_style_coalesce_expression = true:error                                    #Prefer var v = x ?? y; Over var v = x != null ? x : y;
+dotnet_style_collection_initializer = true:error                                 #Prefer var list = new List<int> { 1, 2, 3 }; Over var list = new List<int>(); list.Add(1);
+dotnet_style_explicit_tuple_names = true:error                                   #Prefer (string name, int age) customer = GetCustomer(); var name = customer.name; Over var name = customer.Item1;
+dotnet_style_namespace_match_folder = true:error
```

Comment:
Hmmm, I've seen this rule in another place and it doesn't seem to be related to expressions... Should it be here? 🤔

Reply:
Yes, it was in the today's PR. Visual Studio put it into expressions section... Not me!)